docs/getting-started/integrations-cdn/vuejs-v3.md

```diff
@@ -8,150 +8,114 @@ order: 50
 
-<info-box hint>
-	Starting from version 5.0.0 of this package, you can use native type definitions provided by CKEditor&nbsp;5. Check the details about {@link getting-started/setup/typescript-support TypeScript support}.
-</info-box>
+This guide assumes that you already have a Vue project. If you do not have one, see the [Vue documentation](https://vuejs.org/guide/quick-start) to learn how to create it.
 
 ## Quick start
 
 {@snippet getting-started/use-builder}
 
-### Setting up the project
-
-This guide assumes that you already have a Vue project. If you do not have one, see the [Vue documentation](https://vuejs.org/guide/quick-start) to learn how to create it.
-
-### Installation
-
-Start by installing the following packages:
-
-* `ckeditor5` &ndash; contains all open-source plugins and features for CKEditor&nbsp;5.
-
-	```bash
-	npm install ckeditor5
-	```
-
-* `ckeditor5-premium-features` &ndash; contains premium plugins and features for CKEditor&nbsp;5. Depending on your configuration and chosen plugins, you might not need it.
-
-	```bash
-	npm install ckeditor5-premium-features
-	```
-
-* `@ckeditor/ckeditor5-vue` &ndash; the [CKEditor&nbsp;5 WYSIWYG editor component for Vue](https://www.npmjs.com/package/@ckeditor/ckeditor5-vue).
-
-	```bash
-	npm install @ckeditor/ckeditor5-vue
-	```
-
-With these packages installed, you now need to choose whether to install the `<ckeditor>` component globally or locally and follow the appropriate instructions below.
-
-#### Installing the `<ckeditor>` component globally
-
-To register the `<ckeditor>` component globally, you must install the CKEditor&nbsp;5 plugin for Vue.
-
-If you are using a plain Vue project, you should find the file where the `createApp` function is called and register the `CkeditorPlugin` plugin with the [`use()` method](https://vuejs.org/api/application.html#app-use).
+### Installing and configuring the Vue integration
 
-```js
-import { createApp } from 'vue';
-import { CkeditorPlugin } from '@ckeditor/ckeditor5-vue';
-import App from './App.vue';
+Start by installing the Vue integration for CKEditor&nbsp;5 from npm:
 
-createApp( App )
-	.use( CkeditorPlugin )
-	.mount( '#app' );
+```bash
+npm install @ckeditor/ckeditor5-vue
 ```
 
-If you are using Nuxt.js, you can follow the [Nuxt.js documentation](https://nuxt.com/docs/guide/directory-structure/plugins#vue-plugins) to get access to the `use()` method and register this plugin.
-
-Now you can use the `<ckeditor>` component in any of your Vue components. The following example shows a single file component with open source and premium plugins.
+Once the integration is installed, create a new Vue component called `Editor.vue`. It will use the `useCKEditorCloud` helper to load the editor code from CDN and the `<ckeditor>` component to run it. The following example shows a single file component with open source and premium CKEditor&nbsp;5 plugins.
```

Comment:
```suggestion
### Configuring the integration

Once the integration is installed, create a new Vue component called `Editor.vue`. It will use the `useCKEditorCloud` helper to load the editor code from CDN and the `<ckeditor>` component to run it. The following example shows a single file component with open-source and premium CKEditor&nbsp;5 plugins.
```